build-tools/build-sets

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env node
+'use strict';
+
+const fs = require("fs");
+const child_process = require('child_process');
+const path = require("path");
+
+process.stdout.write("Importing sets from @pokemon-showdown/sets... ");
+
+const shell = cmd => child_process.execSync(cmd, {stdio: 'inherit', cwd: path.resolve(__dirname, '../..')});
+shell(`npm install --no-save @pokemon-showdown/sets`);
+
+const src = path.resolve(__dirname, '../node_modules/@pokemon-showdown/sets');
```

Comment:
Wait have we actually published `@pokemon-showdown/sets`?

Reply:
Yes. 0.0.1